### PR TITLE
Change HAP defaults to be in-line with K8s default settings

### PR DESCRIPTION
**Pull request by coufalja**
In order to make HPA less aggressive

### CODE REVIEW

File: aws/cluster/variables.tf [hunk 1]
```diff
@@ -168,21 +168,21 @@ variable "hpa-sync-period" {
   type        = "string"
   description = "The frequency at which HPA are evaluated and reconciled"
 
-  default = "10s"
+  default = "30s"
```
Comment (coufalja) [inline aws/cluster/variables.tf]: 10s does not make sense as metrics-server scrapes once per 30s

File: aws/cluster/variables.tf [hunk 2]
```diff
@@ -168,21 +168,21 @@ variable "hpa-sync-period" {
...
 }
 
 variable "hpa-scale-down-delay" {
   type        = "string"
   description = "After a downscale, wait at least for this duration before the next downscale"
 
-  default = "1m"
+  default = "5m"
```
Comment (coufalja) [inline aws/cluster/variables.tf]: 1m is too aggressive 5m is K8s default

File: aws/cluster/variables.tf [hunk 3]
```diff
@@ -168,21 +168,21 @@ variable "hpa-sync-period" {
...
 }
 
 variable "hpa-scale-up-delay" {
   type        = "string"
   description = "After an upscale, wait at least for this duration before the next downscale"
 
-  default = "30s"
+  default = "3m"
```
Comment (coufalja) [inline aws/cluster/variables.tf]: 30s is too aggressive 3m is K8s default

File: aws/cluster/variables.tf [hunk 4]
```diff
@@ -168,21 +168,21 @@ variable "hpa-sync-period" {
...
   type        = "string"
   description = "After a downscale, wait at least for this duration before the next downscale"
 
-  default = "1m"
+  default = "5m"
 }
 
 variable "hpa-scale-up-delay" {
   type        = "string"
   description = "After an upscale, wait at least for this duration before the next downscale"
```
Comment (jsedlace) [inline aws/cluster/variables.tf]: not part of this PR, but the description is same as for hpa-scale-down-delay, but shouldn't be
